src/assets/LESSAsset.js

```diff
@@ -33,7 +33,7 @@ class LESSAsset extends Asset {
     return [
       {
         type: 'css',
-        value: this.ast.css,
+        value: this.ast ? this.ast.css : '',
```

Comment:
You could fallback to `this.contents` instead of an empty string, it would end up being the same though. Might be faster as you have it right now.
Would only be really useful if ast can ever be undefined/null while content is not.